proposals/ard_surface_reflectance.json

```diff
@@ -0,0 +1,150 @@
+{
+    "id": "ard_surface_reflectance",
+    "summary": "CARD4L compliant Surface Reflectance generation",
+    "description": "...",
+    "categories": [
+        "cubes",
+        "sar",
+        "ard"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "The source data cube containing multi-spectral optical top of the atmosphere (TOA) reflectances.",
+            "name": "data",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "description": "The digital elevation model to use, leave empty to allow the back-end to make a suitable choice.",
+            "name": "elevation_model",
+            "optional": true,
+            "default": null,
+            "schema": [
+                {
+                    "type": "string",
+                    "subtype": "collection-id"
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        {
+            "description": "The minimum probability required for atmospheric disturbances like clouds for being removed (greater-than-equal).",
+            "name": "threshold",
+            "optional": true,
+            "default": 0.5,
+            "schema": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            }
+        },
+        {
+            "description": "The name of the band dimension.",
```

Comment:
Not sure if users will understand what this is about. Would it be possible to actually avoid the reduce, and use apply instead? If cloud_detection only returns one band, then why the reduce?

Reply:
We probably need to better document/describe it. By default it's bands and is not required to be set and works if the back-end follows the recommended dimension names. It's an edge case we are handing here or we need to add a constraint that the input data cube has a dimension with name or of type bands. I'm fine with either way.

Reply:
Removed for now. With the process graph removed it's not required any longer and the description just says a single dimension of type bands is required.